.github/workflows/nl-action.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       run: |
         if [ -f requirements.txt ]; then pip3 install --upgrade --upgrade-strategy eager -r requirements.txt -e . ; fi
         pip3 install https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
+        pip3 install https://github.com/explosion/spacy-models/releases/download/zh_core_web_sm-3.0.0/zh_core_web_sm-3.0.0.tar.gz
```

Comment:
Maybe it's better to remove here and put it as a prerequisite in the README file, otherwise it might add burden for other PR.

Reply:
I'm not totally sure I understand what you are suggesting. What I am understanding is: you want me to remove the `zh_core_web_sm` model installation from the CI workflow, and write something about installing it manually in my transformations `README.md`. Won't that cause the PR workflow to fail, though?

Reply:
Maybe you could first check the CI results, if it all test cases are passed, then remove it from the workflow file.
Another way is to run `pytest -s --t=pinyin` on your local environment.

Also tagging @kaustubhdhole here.